### DISCUSSION TITLE
关于Modal的静态方法

### DISCUSSION THREAD

**Discussion by wangwufan** [ant-design/ant-design#49702] [Q&A] [answered]
请问Modal.confirm方法的onOk里如何阻止关闭弹窗？我试了return true或false都会关闭。我使用antd 5.19.0

**Comment by Wxh16144** [comment]
印象中没有这个 feature 吧🤔

**Comment by wanpan11** [comment]
没有这个功能 之前是有个 bug https://github.com/ant-design/ant-design/issues/49034 
可以换个思路实现用 `okButtonProps` 来禁用确认点击 🧐

**Comment by linxianxi** [comment, marked as answer]
return Promise.reject()

**Reply by wangwufan** [reply]
谢谢，这个确实可以，只是官网文档没有专门实例介绍
![image](https://github.com/ant-design/ant-design/assets/12657738/aca0e515-ae1f-44d2-9c64-f8edf1b5de43)
